requirements/requirements.txt

```diff
@@ -2,11 +2,11 @@
 # see: https://github.com/boto/botocore/commit/e87e7a745fd972815b235a9ee685232745aa94f9
 python-dateutil==2.8.0
 
-boto3==1.*
-holidays==0.9.*
-matplotlib==3.*
-numpy==1.14.*
-pandas>=0.25.0
-pydantic==0.28.*
-tqdm>=4.23.0
-ujson>=1.35
+boto3>=1.0,<2
+holidays>=0.9,<0.10
+matplotlib>=3.0,<4
+numpy>=1.14,<2
+pandas>=0.25,<0.26
```

Comment:
Do we have issues with 0.26?

Reply:
I think this is OK for now: pandas 0.26 does not exist yet, and we can bump this manually when it comes out